rqd/rqd/rqlog.py

```diff
@@ -0,0 +1,39 @@
+import json
+import logging.config
+import os
+import sys
+import traceback
+
+import rqconstants
+import logs.configs
+
+import ast
+
+
+def setupLogging():
+    """Sets up the logging for RQD.
+       Logs to /var/log/messages"""
+
+    # Always setup the default logging for console and syslog
+    logging.config.dictConfig(logs.configs.DEFAULT_LOGGING)
+
+    if os.path.isfile(rqconstants.LOGGING_CONFIG_FILE):
+        
+        config = None
+        logging.info('Configuring logging from config file %s' % (rqconstants.LOGGING_CONFIG_FILE))
```

Comment:
It'd be great if any new code could follow the PY3 idiom so the `{}` style of string substitution, and double quotes are the default for string literals (to appease the flake8 and [black](https://github.com/psf/black) gods)? Maybe the lads have a different view on that though.

But actually while I'm here - is `logging.info` in scope here?
```suggestion
        logging.info("Configuring logging from config file {}".format(rqconstants.LOGGING_CONFIG_FILE))
```
Thanks Christian - generally this all looks good. I won't get a chance to spin up OpenCue again for a few days but I'm looking forward to playing with this.